static/docs/user-guide/install/linux.md

```diff
@@ -0,0 +1,104 @@
+# Installation on Linux
+
+## Install with pip
+
+```dvc
+$ pip install dvc
+```
+
+Depending on the type of the
+[remote storage](/doc/user-guide/external-dependencies) you plan to use, you
+might need to install optional dependencies: `[s3]`, `[ssh]`, `[gs]`, `[azure]`,
+and `[oss]`. Use `[all]` to include them all.
+
+<details>
+
+### Example: How to install DVC with support for Amazon S3 storage
+
+```dvc
+$ pip install 'dvc[s3]'
+```
+
+In this case it installs `boto3` library as well, besides DVC.
+
+</details>
+
+## Install with conda
+
+> Requires [Anaconda Distribution](https://docs.anaconda.com/anaconda/).
+
+```dvc
+$ conda install -c conda-forge dvc
+```
+
+> Currently, it supports only Python versions 2.7, 3.6, and 3.7.
+
+## Install from repo
```

Comment:
repo -> repository? I'm not sure what is the reason to cut it here. If take a look at Ubuntu's docs they don't use "repo" term as far as I can tell.

Reply:
This is how it is called in the url: `https://dvc.org/rpm/dvc.repo`. Anyway, fixed it.

Reply:
I had addressed this in 1774355.